index.html

```diff
@@ -1,23 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+  <link rel='stylesheet' href='style-sheet.css'>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Tic Tac Toe</title>
 </head>
+
 <body>
-  <h1>Tic Tac Toe!</h1>
+  <div id='header-container'>
+  <h1>TIC TAC TOE</h1>
+</div>
+<div id='player-wrapper'>
+<div id='player-container'>
+  <form id='Player-vs-Comp-selector'>
+    <div class='player-selector-container'>
+    <input id='Player-vs-Player' name='game-mode' type='radio' value="PvP" checked="checked">
+    <label for='Player-vs-Player' name='game-mode'>Player vs Player</label><br>
+  </div>
+  <p id="timer"></p>
+  <div class='player-selector-container'>
+    <input id='Player-vs-Comp' name='game-mode' type='radio' value="PvC">
+    <label for='Player-vs-Comp' name='game-mode' type='radio'>Player vs Computer</label></div>
+  </form>
+</div>
+</div>
+<div id='game-wrapper'>
+<div id="main-game-container">
+  <div>
+    <div class='name-field-container'>
+    <label for="x-player">X Player:</label>
+    <input type="text" id="x-player" placeholder="Name">
+  </div>
+  <div class='name-field-container'>
+    <label for="o-player">O Player:</label>
+    <input type="text" id="o-player" placeholder="Name">
+  </div>
+  </div>
+
+  <div id='status-text'></div>
+
   <div id='board'>
-    <div class='cell'></div>
-    <div class='cell'></div>
-    <div class='cell'></div>
-    <div class='cell'></div>
-    <div class='cell'></div>
-    <div class='cell'></div>
-    <div class='cell'></div>
-    <div class='cell'></div>
-    <div class='cell'></div>
+    <div class='cell' id='1'></div>
+    <div class='cell' id='2'></div>
+    <div class='cell' id='3'></div>
+    <div class='cell' id='4'></div>
+    <div class='cell' id='5'></div>
+    <div class='cell' id='6'></div>
+    <div class='cell' id='7'></div>
+    <div class='cell' id='8'></div>
+    <div class='cell' id='9'></div>
+  </div>
+<div id='button-container'>
+  <div class='button-wrapper'>
+    <button id='start'>Start Game</button>
+    
+  </div>
+  <div class='buttonwrapper'>
+    <button id='new-game'>New Game</button>
```

Comment:
You have some great comments throughout your CSS and JS files, don't forget the HTML page. They don't need to be comments necessarily, but at least some organizational tags like Game Play Status, Name Input  Container, Game Board, etc.